editor/plugins/texture_editor_plugin.cpp

```diff
@@ -124,25 +149,50 @@ void TexturePreview::_update_metadata_label_text() {
 }
 
 TexturePreview::TexturePreview(Ref<Texture2D> p_texture, bool p_show_metadata) {
+	set_custom_minimum_size(Size2(0.0, 256.0) * EDSCALE);
+
+	bg_rect = memnew(ColorRect);
+	bg_rect->set_anchors_preset(PRESET_FULL_RECT);
```

Comment:
Actually TexturePreview is a container itself, so all anchor presets can be removed.